docker/vm/nodejs/nginx_vm.conf

```diff
@@ -0,0 +1,82 @@
+upstream django_app {
+  server django:8000 fail_timeout=0;
+}
+
+# Django server
+server {
+  # Listen on port 80 & then redirect to SSL port
+  listen 80;
+  server_name BACKEND_DOMAIN;
+
+  # redirection to SSL 
+  location / {
+    return 301 https://$host$request_uri;
+  }
+
+  # Certbot challenge files
+  location /.well-known/acme-challenge/ {
+    root /var/www/certbot;
+  }
+}
+
+server {
+  listen 443 ssl;
+  server_name BACKEND_DOMAIN;
+  client_max_body_size 400M;
+
+  # SSL certificates generated by init-letsencrypt
+  ssl_certificate /etc/letsencrypt/live/DOMAIN_NAME/fullchain.pem;
```

Comment:
Why do we have to replace the paths for ssl certs from lets encrypt? Can't this be placed at a fixed location say `/etc/certs/frontend/` and `/etc/certs/backend`?

Reply:
Already answered in different comment